src/test/java/com/pubnub/api/models/consumer/history/PNFetchMessageItemTest.java

```diff
@@ -19,33 +17,70 @@ void when_includeMessageType_is_false_then_getMessageType_should_return_null() {
         boolean includeMessageType = false;
         String spaceId = null;
         Integer pnMessageType = null;
-        String userMessageType = null;
+        String type = "myType";
 
-        objectUnderTest = getPNFetchMessageItem(includeMessageType, spaceId, pnMessageType, userMessageType);
+        objectUnderTest = getPNFetchMessageItem(includeMessageType, spaceId, pnMessageType, type);
 
         assertNull(objectUnderTest.getMessageType());
+        assertEquals(type, objectUnderTest.getType());
```

Comment:
Why test type if test name is about messageType?

Reply:
Ok, I can create separate test to cover this case.